changes-set.txt

```diff
@@ -26,6 +26,43 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+17-Sep-23 cdaun     endjudisj   Changes from +c notation to |_|
+17-Sep-23 cdaen     djuen       Changes from +c notation to |_|
+17-Sep-23 cdafn     ---         |_| is not a function
+17-Sep-23 cdaenun   djuenun     Changes from +c notation to |_|
+17-Sep-23 cdadom1   djudom1     Changes from +c notation to |_|
+17-Sep-23 cdacomen  djucomen    Changes from +c notation to |_|
+17-Sep-23 cdadom2   djudom2     Changes from +c notation to |_|
+17-Sep-23 pwxpndom2cda pwxpndom2 Changes from +c notation to |_|
+17-Sep-23 xp2cda    xp2dju      Changes from +c notation to |_|
+17-Sep-23 uncdadom  undjudom    Changes from +c notation to |_|
+17-Sep-23 mapcdaen  mapdjuen    Changes from +c notation to |_|
+17-Sep-23 cdadom3   djudoml     Changes from +c notation to |_|
```

Comment:
Both cdadom3 and cdadom1 are replaced with djudom1.  Is this ok?

Reply:
The replacement for cdadom3 is `djudoml` (with the letter l) and the replacement for cdadom1 is `djudom1` (with the digit 1).

Reply:
Thanks for the explanation.